### PR TITLE
Completed es6-template-literals.

### PR DESCRIPTION
**Pull request by saraysc**
image

### CODE REVIEW

**Comment by saraysc** [comment]
image

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze: ' + address);
```
Comment (Zhiwutian) [inline es6-template-literals/index.html:15]: you should be using `,` here instead of `+`

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio: ' + bio);
```
Comment (Zhiwutian) [inline es6-template-literals/index.html:23]: you should be using `,` here instead of `+`

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log('I have a ' + item);
```
Comment (Zhiwutian) [inline es6-template-literals/index.html:30]: you should be using `,` here instead of `+`

File: es6-template-literals/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    console.log('I have an ' + fruit);
```
Comment (Zhiwutian) [inline es6-template-literals/index.html:31]: you should be using `,` here instead of `+`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one final update for the console logs!

File: es6-template-literals/index.html (hunk 4 above)
Comment (saraysc) [reply es6-template-literals/index.html]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**